yt_dlp/extractor/vimeo.py

```diff
@@ -239,24 +250,39 @@ def _parse_config(self, config, video_id):
             '_format_sort_fields': ('quality', 'res', 'fps', 'hdr:12', 'source'),
         }
 
-    def _call_videos_api(self, video_id, jwt_token, unlisted_hash=None, **kwargs):
+    def _fetch_oauth_token(self):
+        if not self._ios_oauth_token:
+            self._ios_oauth_token = self.cache.load(self._NETRC_MACHINE, self._IOS_OAUTH_CACHE_KEY)
+
+        if not self._ios_oauth_token:
+            self._ios_oauth_token = self._download_json(
+                'https://api.vimeo.com/oauth/authorize/client', None,
+                'Fetching OAuth token', 'Failed to fetch OAuth token',
+                headers={
+                    'Authorization': f'Basic {self._IOS_CLIENT_AUTH}',
+                    **self._IOS_CLIENT_HEADERS,
+                }, data=urlencode_postdata({
+                    'grant_type': 'client_credentials',
+                    'scope': 'private public create edit delete interact upload purchased stats',
+                }, quote_via=urllib.parse.quote))['access_token']
```

Comment:
Apparently, the default `urllib.parse.quote_plus` also works.

Reply:
Just following what the app does

Reply:
Better to do that, but Pythons < 3.5 need a hack to implement `quote_via` (`urllib.urlencode()` is already compat-ed for Py2 in _yt-dl_, so not a big deal), which is why I noticed.